.tekton/osc-operator-pull-request.yaml

```diff
@@ -7,8 +7,10 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "devel"
+    pipelinesascode.tekton.dev/on-cel-expression:
+      event == "pull_request" &&
+      target_branch == "devel" &&
+      files.all.exists(path, !path.matches('.tekton/.*fbc-.*|fbc/.*))
```

Comment:
Do we need to do the same for the other pipelines?

Reply:
no, the other pipelines start only when their files are modified ("allow list")

only the operator pipeline uses a "deny list"